inspirehep/logger/config.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from logging.config import dictConfig
+
+SENTRY_DSN = None
+"""
+The DSN tells the SDK where to send the events to.
+"""
+
+SENTRY_SEND_DEFAULT_PII = False
+"""
+If this flag is enabled, certain personally identifiable information is added by active integrations.
+Without this flag they are never added to the event, to begin with.
+"""
+
+# Configuration for logging, deafault level is ``ERROR``
+dictConfig(
+    {
+        "version": 1,
+        "formatters": {
+            "default": {
+                "format": "[%(asctime)s: %(levelname)s/%(processName)s] %(name)s: %(message)s"
+            }
+        },
+        "handlers": {
+            "stdout_handler": {"class": "logging.StreamHandler", "formatter": "default"}
```

Comment:
don't we want to log more on stdout (like WARNING/INFO)?

Reply:
updated

Reply:
is it what you meant? about the ``level``?

Reply:
yeah, but don't you need to set the priority fo sentry too? or is that automatic?

Reply:
it's automatic for sentry